frontend/app/views/spree/shared/_nav_bar.html.erb

```diff
@@ -18,35 +18,37 @@
       </div>
     </div>
   </li>
-  <li data-hook>
-    <div class="dropdown navbar-right-dropdown">
-      <button type="button" data-toggle="dropdown" class="navbar-right-dropdown-toggle" aria-label="<%= Spree.t('nav_bar.show_user_menu')%>">
-        <%= icon(name: 'person',
-                 classes: 'd-none d-xl-inline-block',
-                 width: 41.4,
-                 height: 36)  %>
+  <% if defined?(account_link_path) %>
```

Comment:
`spree_auth_devise` gem might not be enabled in a consumer app, so we have to check it here